cobald_tests/daemon/test_service.py

```diff
@@ -133,3 +134,22 @@ async def co_pingpong(what=default):
                     break
             else:
                 assert len(reply_store) == 9
+
+    @pytest.mark.parametrize("flavour", (asyncio, trio))
```

Comment:
Is there a reason to exclude "threading" here? See same on `test_interrupt` below.

Reply:
Not that I could think of. Seems like either I c/p'ed this erroneously, or should document it. ^^

Reply:
The issue was that ``async`` uses separate methods from ``threading``. I've parameterized them now and added threading.